drivers/clock_control/CMakeLists.txt

```diff
@@ -26,6 +26,9 @@ zephyr_library_sources_ifdef(CONFIG_CLOCK_CONTROL_NRF                 clock_cont
 zephyr_library_sources_ifdef(CONFIG_CLOCK_CONTROL_NRF_DRIVER_CALIBRATION nrf_clock_calibration.c)
 zephyr_library_sources_ifdef(CONFIG_CLOCK_CONTROL_RV32M1_PCC          clock_control_rv32m1_pcc.c)
 zephyr_library_sources_ifdef(CONFIG_CLOCK_CONTROL_INFINEON_CAT1       clock_control_ifx_cat1.c)
+zephyr_library_sources_ifdef(CONFIG_CLOCK_CONTROL_IFX_CAT1_FIXED_CLOCK       clock_control_ifx_cat1_fixed_clock.c)
```

Comment:
Do we need 3 different drivers? I will check the clock tree, could you please move this commit to base.